cmd/loki/Dockerfile

```diff
@@ -7,20 +7,21 @@ WORKDIR /src/loki
 RUN make clean && (if [ "${TOUCH_PROTOS}" ]; then make touch-protos; fi) && make BUILD_IN_CONTAINER=false loki
 
 FROM alpine:3.9
-RUN apk add --update --no-cache ca-certificates libcap \
```

Comment:
I think there's a tradeoff here of caching vs layer reduction and I'm inclined to think that caching is a bit more preferrable although I'm biased by building this regularly. That being said, I don't think it's a huge deal either way.

Reply:
This needs to stay, see below

Reply:
sry deleted my last message, didn't read closely enough

Reply:
I think I'd like to leave this as is, same as the changes below for 2 reasons:

1. We spent a stupid amount of time trying to test this worked after adding this change and I don't have time to redo this.
2. I prefer the separate run steps below I think it's easier to read and understand what the steps are doing/for

Reply:
Could you elaborate on what you spent a lot of time testing? Adding a new package should be as simple as adding an `apk add --no-cache <pkgname>`.

Reply:
All the cap stuff was new and we tried several combinations of where to install `libcap` as well as removing it and where to remove it to keep the resulting image size as small as possible as well as make sure it still works.